content/stx-stacking/delegation/_index.md

```diff
@@ -48,14 +48,16 @@ Delegating (or pooling) STX overcomes both issues above because the total STX lo
 
 ### How do I select a pool?
 
-A list of pool providers can be found on both [stacks.co](https://www.stacks.co/stacking) and [stacks.org](https://www.stacks.org/stacking).
+A list of pool providers can be found on [stacks.co](https://www.stacks.co/stacking) and the [Pool Registry]([https://www.stacks.org/stacking](https://stacks-pool-registry.pages.dev)).
```

Comment:
The links got a bit jumbled here, updated stacks.co to the final page (it redirects to /learn) and fixed the pool registry link.

```suggestion
A list of pool providers can be found on [stacks.co](https://www.stacks.co/learn/stacking) and the [Pool Registry](https://stacks-pool-registry.pages.dev).
```